tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/CppSymbolVisitor.java

```diff
@@ -5,6 +5,7 @@
 
  package com.amazonaws.util.awsclientsmithygenerator.generators;
 
+ import lombok.val;
```

Comment:
Dont use Lombok val, spell out the complete type

Reply:
ok, that was IDE suggestion. will change